### PR TITLE
migrated schedular to structured logging

### PR DESCRIPTION
**Pull request by shivanshuraj1333**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Marking files of `pkg/scheduler` and `cmd/kube-scheduler` as migrated to structured logging
#### Which issue(s) this PR fixes:

Reference https://github.com/kubernetes/kubernetes/issues/105841

#### Special notes for your reviewer:
This is not complete migration of `pkg/scheduler` and `cmd/kube-scheduler`, the complete migration is blocked by https://github.com/kubernetes/kubernetes/issues/106262 and https://github.com/kubernetes/kubernetes/issues/106276
after providing appropriate fixes for the mentioned issues we need to update the packages here as `pkg/scheduler/...` and `cmd/kube-scheduler/...`

Excluded files to achieve the complete migration of scheduler are `cmd/kube-scheduler/app/options/configfile.go` line-96
,and `pkg/scheduler/internal/cache/debugger/dumper.go` line-49, line-60

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by serathius** [comment]
Got `logcheck: -: named files must be .go files: /home/prow/go/src/k8s.io/kubernetes/cmd/kube-proxy/...` not sure what's the cause and how it can be broken on master.

File: hack/.structured_logging [hunk 1]
```diff
@@ -1,4 +1,40 @@
 cmd/kube-proxy/...
+cmd/kube-scheduler/app/config/...
+cmd/kube-scheduler/app/options/...
+cmd/kube-scheduler/app/server_test.go
```
Comment (serathius) [inline hack/.structured_logging:4]:
```suggestion
```

**Review by pohly — APPROVED** [review]
/lgtm
/retest

**Comment by serathius** [comment]
/milestone v1.23
This PR blocks https://github.com/kubernetes/kubernetes/issues/105841

**Comment by shivanshu1333** [comment]
pinging @dims for approval

**Pull request merged**